src/nvme/fabrics.c

```diff
@@ -357,10 +357,18 @@ static int __add_argument(char **argstr, const char *tok, const char *arg)
 	return 0;
 }
 
+static int __nvmf_supported_options(nvme_root_t r);
+#define nvmf_check_option(r, tok)					\
+({									\
+	if (!(r)->options)						\
```

Comment:
This makes the `if (r->options) return 0;` check in `__nvmf_supported_options()` redundant now?